src/main/java/winstone/AbstractSecuredConnectorFactory.java

```diff
@@ -100,12 +100,19 @@ protected SslContextFactory.Server getSSLContext( Map<String, String> args) {
             ssl.setKeyManagerPassword(privateKeyPassword);
             ssl.setKeyManagerFactoryAlgorithm(Option.HTTPS_KEY_MANAGER_TYPE.get(args));
             ssl.setCertAlias(Option.HTTPS_CERTIFICATE_ALIAS.get(args));
-            ssl.setExcludeProtocols("SSLv3", "SSLv2", "SSLv2Hello");
+            String excludeProtos = Option.HTTPS_EXCLUDE_PROTOCOLS.get(args);
```

Comment:
Good catch, fixed

Reply:
Other options with a default value (like `HTTPS_KEY_MANAGER_TYPE`, `SIMPLE_ACCESS_LOGGER_FORMAT`, and `SIMPLE_ACCESS_LOGGER_FILE`), use the two-argument version of an option declaration that accepts a default value as the second argument. I think it would be more consistent with existing options to encode the default there in contrast to your current approach which has no default but then encodes the default at the consumer.

Reply:
Thanks have set it using defaultValue now, let me know if it looks right